.gitignore

```diff
@@ -1,32 +1,45 @@
-*~
-.DS_Store
+# Clojure / leiningen files
+pom.xml
+pom.xml.asc
+*.jar
+*.class
+/lib/
+/classes/
+/target/
+/checkouts/
 .lein-deps-sum
 .lein-repl-history
+.lein-plugins/
+.lein-failures
 .nrepl-port
-/pom.xml
-/pom.xml.asc
-/clojush*.jar
-classes/
-target/
+.cpcache/
+.lein-project-checksum
+
+# Editor and backup files
+*~
+.idea
+.gorilla-port
+.ipynb_checkpoints/
+.calva/
+.clj-kondo/
+.lsp
+
+# OS Files
+.DS_Store
+
+# Results and data directories
 results/
+/data/
+logs/
+
+# Unknown / Other
 .classpath
 .project
 .settings/ccw.repl.cmdhistory.prefs
-/data/
-logs/
-lib/
 /bin
-.lein-failures
 doc/
 cov/*
 out
```

Comment:
This is sometimes the name of the folder which the compiled classes/jars get put. Similar to `target/`, which is already ignored above. I would recommend picking one and having all the tools dump there files there.